CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # 2020-01-08
 
+## API changes (relevant for self-hosters and client developers)
+
+- Allow up to 256 characters as handle, dots and dashes too (#953)
+- Refuse to delete non-empty IdPs (412 precondition failed) (#875)
+- Make sure that someone is SSO user before setting ManagedBy (#947)
```

Comment:
these two look like bug fixes / internal changes to me.

Reply:
I guess we can debate this, my thinking here was: executing the same request against our API now returns a different result. This was not something that was there by design, we _changed_ the design. WDYT?

Reply:
Sure, leave them there, but maybe write which endpoints now return different results. `/endpoint1 now behaves differently (#PRnumber)` is more informational for client developers looking at the changelog I think.